src/Adapter/Carrier/QueryHandler/GetAvailableCarriersHandler.php

```diff
@@ -101,33 +129,65 @@ public function handle(GetAvailableCarriers $query): GetCarriersResult
     }
 
     /**
-     * Compute intersection of carriers across all products.
+     * @param ProductQuantity[] $productQuantities
+     *
+     * @return Product[]
+     */
+    private function loadProducts(array $productQuantities): array
+    {
+        $products = [];
+        foreach ($productQuantities as $pq) {
+            $productId = $pq->getProductId();
```

Comment:
```suggestion
        foreach ($productQuantities as $productQuantity) {
            $productId = $productQuantity->getProductId();
```
It doesn't cost much to have a more meaningful name.